sbt-scala-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePluginInternal.scala

```diff
@@ -411,7 +425,8 @@ object ScalaNativePluginInternal {
       else binaryName
 
     sys.env.get(s"${envName}_PATH") match {
-      case Some(path) => file(path)
+      case Some(path) =>
+        file(if (isWindows) path.replaceAll("clang-cl", binaryName) else path)
```

Comment:
Yeah, I will fix it, I just don't want to reboot a computer before finish exception handler